docker/test/fuzzer/run-fuzzer.sh

```diff
@@ -174,7 +174,7 @@ function fuzz
     mkdir -p /var/run/clickhouse-server
 
     # NOTE: we use process substitution here to preserve keep $! as a pid of clickhouse-server
-    clickhouse-server --config-file db/config.xml --pid-file /var/run/clickhouse-server/clickhouse-server.pid -- --path db > server.log 2>&1 &
+    clickhouse-server --config-file db/config.xml --pid-file /var/run/clickhouse-server/clickhouse-server.pid -- --path db > server.log 2>>stderr.log &
```

Comment:
Not sure what I did wrong (I did something similar to stress tests) but all the logs are now in stderr.log and nothing in server.log. The idea was to have logs and sanitizer reports separated. Need to check why it didn't happen